CONTRIBUTING.adoc

```diff
@@ -114,44 +137,76 @@ Changes to the XML schema will be overwritten by the Gradle build task.
 
 = Squash commits
 
-Use git rebase –interactive, git add –patch and other tools to "squash" multiple commits into atomic changes. In addition to the man pages for git, there are many resources online to help you understand how these tools work. Here is one: https://book.git-scm.com/4_interactive_rebasing.html[https://book.git-scm.com/4_interactive_rebasing.html].
+Use `git rebase --interactive`, `git add --patch` and other tools to "squash" multiple commits into atomic changes. In addition to the man pages for git, there are many resources online to help you understand how these tools work. Here is one: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History[https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History].
```

Comment:
Will your please change

```
pages for git
```

to

```
pages for `git`
```

Reply:
Will you please move the link to be on "many resources online" instead of having a separate sentence? Then also please remove the "Here is one" sentence.

Reply:
Done.